drupal/sites/default/boinc/modules/boincuser/includes/boincuser.forms.inc

```diff
@@ -727,14 +727,14 @@ function boincuser_optinform() {
 function boincuser_optinform_validate($form, &$form_state) {
   // Check opt-in
   if (!$form_state['values']['optin']) {
-    form_set_error('legaloptin', bts('ERROR: You must acknowledge our terms of use by clicking the checkbox before registering for an account.', array(), NULL, 'boinc:opt-in-form'));
+    form_set_error('legaloptin', bts('ERROR: You must acknowledge our terms of use by clicking the checkbox before registering for an account.', array(), NULL, 'boinc:termsofuse-form'));
   }
 }
 
 function boincuser_optinform_submit($form, &$form_state) {
   $user = $form['#account'];
   if (!boincuser_consentto_termsofuse($user)) {
-    form_set_error('legaloptin', bts('There was an error in the opt-in process. Please contact the site administrators.', array(), NULL, 'boinc:opt-in-form'));
+    form_set_error('legaloptin', bts('There was an error in the opt-in process. Please contact the site administrators.', array(), NULL, 'boinc:termsofuse-form'));
```

Comment:
Is the remaining "opt-in" (process) intentional?

Reply:
No, sorry. I've changed it in the latest commit.

Reply:
No worries, that's what reviews are for.